unitTestSupportFunctions/calculateForce.m

```diff
@@ -0,0 +1,51 @@
+function [Fx, Fy, Fz] = calculateForce (FComplete, fldMsh, homDBC, parameters, noDimensions)
```

Comment:
As mentioned before, this function will be from now on user defined and be placed under a new directory in FEMComputationalFluidDynamicsAnalysis with the name postProcessing. Just to be more clear, the function name and implementation should be defined by the user. The function name should then be appropriately written out to the GiD generated input file and parsed appropriately with your parser. Then, you can use the str2func command in Matlab to convert the parsed string into an actual Matlab function which can then be used to compute the desirable postprocessing quantity. The output should be for all such functions a vector F of n components where the number of components is desided by the user depending on what the function is supposed to perform (see also my previous comment above).